### ISSUE TITLE
Compute offerings are inaccessible as regular user.

### ISSUE THREAD

**Issue by vladimirpetrov** [apache/cloudstack#6145]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
API (I think)
~~~

##### CLOUDSTACK VERSION

~~~
4.17
~~~

##### CONFIGURATION

Default config

##### OS / ENVIRONMENT

N/A

##### SUMMARY

Compute offerings are inaccessible as regular user.

##### STEPS TO REPRODUCE

~~~
Just log in as a regular user and try to deploy a VM.
~~~

##### EXPECTED RESULTS

~~~
The regular user is able to access compute offerings.
~~~

##### ACTUAL RESULTS

~~~
The regular user is not able to access compute offerings.
~~~

![image](https://user-images.githubusercontent.com/12384665/159237693-c65120f9-4994-480d-a5fc-6b6cdc40b6f6.png)

According to @weizhouapache, this might be related to #5008

**Comment by weizhouapache** [comment]
@vladimirpetrov 
this issue #6145 seems to be caused by #6045, and has been fixed by #6126 which reverted #6045

**Comment by nvazquez** [comment]
Thanks @weizhouapache for checking
